react-app/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "react-app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "npx webpack --config webpack/webpack.prod.js",
+    "develop": "npx webpack --watch --config webpack/webpack.dev.js",
+    "start": "npx webpack serve --config webpack/webpack.dev.js",
+    "test": "jest",
+    "ts:check": "tsc --noEmit --project ./tsconfig.json"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@babel/core": "^7.12.16",
+    "@babel/plugin-proposal-class-properties": "^7.13.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.13.0",
+    "@babel/plugin-transform-modules-commonjs": "^7.13.0",
+    "@babel/preset-env": "^7.12.16",
+    "@babel/preset-react": "^7.12.13",
+    "@babel/preset-typescript": "^7.13.0",
+    "@fortawesome/fontawesome-svg-core": "^1.2.35",
+    "@fortawesome/free-solid-svg-icons": "^5.15.3",
+    "@fortawesome/react-fontawesome": "^0.1.14",
+    "@types/jest": "^26.0.20",
+    "@types/node": "^14.14.28",
+    "@types/react": "^17.0.2",
+    "@types/react-dom": "^17.0.1",
+    "@types/styled-components": "^5.1.8",
+    "@types/webpack": "^4.41.26",
+    "autoprefixer": "^10.2.4",
+    "babel-jest": "^26.6.3",
+    "babel-loader": "^8.2.2",
+    "clean-webpack-plugin": "^3.0.0",
+    "css-loader": "^5.0.2",
+    "eslint-loader": "^4.0.2",
+    "file-loader": "^6.2.0",
+    "glob": "^7.1.6",
+    "imagemin-webpack-plugin": "^2.4.2",
+    "jest": "^26.6.3",
+    "mini-css-extract-plugin": "^1.3.6",
+    "node-sass": "^5.0.0",
```

Comment:
It's only my personal opinion, but I think, CSS-in-JS and CSS/SASS modules do not play together exceptionally well within the same App. Personally, I would make a decision in favor of either of the two and get rid of the other one.